main-app/api/server/routes/agreements_obligations.js

```diff
@@ -0,0 +1,35 @@
+const passport = require('passport');
+const knex = require('../config/knex_config.js');
+
+const jwtAuth = passport.authenticate('jwt', { session: false });
+
+module.exports = (app) => {
+  app.put('/agreements_obligations/:id', jwtAuth, (req, res) => {
+    knex('agreements_obligations')
+      .where('id', req.params.id)
+      .update(req.body.data, Object.keys(req.body.data))
+      .then(agreements_obligations => res.status(200).send({ agreements_obligations }))
+      .catch(err => res.status(500).send(err));
+  });
+
```

Comment:
i take it we don't need a post endpoint because agreements_obligations belongs to a participant?

Reply:
Yes and i forgot to do that chunk in participants.js. Fixed